### ISSUE TITLE
Homebrew

### ISSUE THREAD

**Issue by magnetardev** [EclipseEmu/eclipse-native#14]
Add support for adding games from homebrew. Pretty much just load https://eclipseemu.me/homebrew.json into a view.

- [ ] List
- [ ] Add
- [ ] Search

**Comment by magnetardev** [comment]
Homebrew list should probably be moved to a new repo, and be available at https://homebrew.eclipseemu.me/

**Comment by Neptune1190** [comment]
would love if homebrew for more then just GBA was added,i know nes has a fairly big scene.

**Comment by magnetardev** [comment]
Yeah I intend on having games for every system, and making it so people can submit their own.

**Comment by magnetardev** [comment]
I'm actually going to put this on hold until after v1, seeing as its not necessary for emulator apps to include. A community maintained-list will be integrated in the future.
